scss/core/abstracts/_variables.scss

```diff
@@ -746,6 +746,9 @@ $badge-padding-y:                   .4em !default;
 $badge-padding-x:                   1em !default;
 $badge-border-radius:               10rem !default;
 
+$badge-notification-border: 2px solid $red-100 !default;
+$badge-notification-color: $red-100 !default;
```

Comment:
@Pallavi-PH, Is badge-notification have only  `$red-100` color ?? As of now.

Reply:
Yes, currently we have only one variation if badge notification.

Reply:
Okay thanks